backend/app/project/routes.py

```diff
@@ -1,8 +1,64 @@
-import uuid
-from flask import jsonify, request, abort
+from flask import Flask, jsonify, request, abort
 from app import db
-from app.models import Project
 from . import project
+from app.models import Project
+import uuid
+
+
+# get all endpoints
+@project.route("", methods=["GET"])
+def get_projects():
```

Comment:
@peterghrong Isn't this done in another pr? If so, is there a need to write it here (aka is there a need to merge #98)?

Reply:
Yes I just pulled from the other branch so everything in this branch is somewhat up to date. Apologies if this is bad practice !